src/Roslyn.Diagnostics.Analyzers/CSharp/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,5 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+### New Rules
+Rule ID | Category | Severity | Notes
```

Comment:
@Evangelink Didn't you fix this before (the empty line before the table)? or I'm mis-remembering?

Reply:
I don't recall doing the fix :)